docs/setup.md

```diff
@@ -62,17 +62,24 @@ You can also contact us on slack for any further queries.
 ## TL;DR bare minimum version
 If you know your way around Rails, here's the very short version. Some additional requirements are necessary to make all the tests pass and all the features work, but this should be enough to stand up the app quickly.
 
-* copy `config/application.example.yml` to `config/application.yml`
-* copy `config/database.example.yml` to `config/database.yml`
-* create a MySQL database, `dashboard`
-* install ruby 2.7.1 and nodejs
-* `bundle install`
-* `rake db:migrate`
-* install yarn
-* `yarn` for more javascript requirements
-* `yarn start` to build assets
-* `guard` or `rails s` to start a server
-* localhost:3000 should load the home page
+1. Fork our repo from https://github.com/WikiEducationFoundation/WikiEduDashboard.
+2. In the console, download a copy of your forked repo with `git clone https://github.com/your_username/WikiEduDashboard.git` where `your_username` is your GitHub     username.
+3. Enter the new WikiEduDashboard directory with `cd WikiEduDashboard`.
```

Comment:
I think just `fork and clone the repo from ...` is more appropriate for `TL;DR version. Additional detail such as the commands to do this, the step of entering the directory afterwards, are more appropriate for the `Detailed instructions` section that follows.